src/mpd.c

```diff
@@ -2066,6 +2066,65 @@ mpd_get_query_params_find(int argc, char **argv, struct query_params *qp)
 }
 
 static int
+mpd_command_count(struct evbuffer *evbuf, int argc, char **argv, char **errmsg)
+{
+  struct query_params qp;
+  struct count_info ci;
+  int ret;
+
+  if (argc < 3 || ((argc - 1) % 2) != 0)
+    {
+      DPRINTF(E_LOG, L_MPD, "Missing argument(s) for command 'find'\n");
+      ret = asprintf(errmsg, "Missing argument(s) for command 'find'");
```

Comment:
There a quite a few of these error message duplications in mpd.c. Would it be possible to omit the DPRINTF here, and then let the caller decide whether to DPRINTF? If you require the log severity and don't want export that, then maybe you could avoid some duplication with a short-hand function?